src/commands/package-and-push-with-abs.yaml

```diff
@@ -12,6 +12,11 @@ parameters:
       When this is `false`, commits to `master` will be pushed to `app_catalog` instead of `app_catalog_test`.
       Set this to `false` for deployments that follow a a master branch for production releases rather than
       using tags (the default).
+  skip_conftest_deprek8ion:
+    type: boolean
+    default: false
+    description: |
+      When this is `false`, checking for deprecated manifest versions will be skipped.
```

Comment:
mmm... it sounds like it should be the other way around? if the parameter is called `skip_conftest_deprek8ion` , shouldn't it be
> When this is `true`, checking for deprecated manifest versions will be skipped

wdyt?

Reply:
yeah :) fixed